test/setup.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+const chai = require('chai');
+
+global.sinon = require('sinon');
+global.assert = chai.assert;
```

Comment:
@narqo care to provide rationale why it should be fixed? :)
We use this technique almost everywhere, it's rather convenient.